js/packages/web/src/components/AuctionCard/index.tsx

```diff
@@ -444,7 +445,8 @@ export const AuctionCard = ({
         bids,
       ).then(async () => {
         await update();
-        setShowRedeemedBidModal(true);
+        setIsOpenCongratulations(canClaimPurchasedItem ? 'claim' : 'purchase');
```

Comment:
Move to constants. Also I don't see use of 'claim' state.

Reply:
'claim' will be used for second modal (instant sale claim)